src/XIVLauncher.Common/Game/Launcher.cs

```diff
@@ -40,7 +40,7 @@ public class Launcher
 
     private const string FALLBACK_FRONTIER_URL_TEMPLATE = "https://launcher.finalfantasyxiv.com/v620/index.html?rc_lang={0}&time={1}";
 
-    public Launcher(ISteam? steam, IUniqueIdCache uniqueIdCache, ISettings settings, string? frontierUrl)
+    public Launcher(ISteam? steam, IUniqueIdCache uniqueIdCache, ISettings settings, string? frontierUrl = null)
```

Comment:
This seems out of scope for this PR

Reply:
Fair enough. It was just in so I could build against unmodified xlcore. Can definitely remove.